File: method.py
class Calculator:
    def __init__(self, val1, val2):
        self.val1 = val1
        self.val2 = val2
    
    # Method defined inside the class
    def add(self, val1, val2):
        print(val1 + val2) 
    
    # Method defined outside the class
    def temp(self, v):
        return v


obj = Calculator(3, 4)


print(obj.temp(5))

File: method.cpp
#include <iostream>
using namespace std;

class Calculator{
    public:
    int val1,val2;

    Calculator(int val1,int val2){
        this->val1 = val1;
        this->val2 = val2;

    }
    

    // inside the function
    int add(int val1,int val2){
        cout<< val1+val2;
    }

    int temp(int v);
};

// outside the class
int Calculator::temp(int v){
    return v;
}

int main(){
    Calculator obj(3,4);
    cout<<endl<<obj.temp(5);
}